Makefile

```diff
@@ -31,15 +31,17 @@ all: build test
 # Downloads EMR packages. Skips if the tar file containing EMR packages has been made.
 
 # Builds and moves container python library into the Docker build context
-build-container-library:
+build-container-library: init
 	python setup.py bdist_wheel;
 	cp dist/*.whl ${BUILD_CONTEXT}
 
 init:
 	pip install pipenv --upgrade
-	pipenv install
+	pipenv lock
+	pipenv install --system
```

Comment:
Is the `--system` flag needed? `pipenv install --system` would bypass a contributor's virtualenv and install into their system python environment, is that right? If so, we shouldn't run this command, since this target could be used by contributors attempting to build this package. Could you remove it if it isn't needed?